common/lib/xmodule/xmodule/course_module.py

```diff
@@ -830,6 +830,15 @@ class CourseFields(object):
         },
         scope=Scope.settings
     )
+    allow_unsupported_xblocks = Boolean(
+        display_name=_("Add Unsupported Problems and Tools"),
+        help=_(
+            "Enter true or false. If true, you can add unsupported content types to your course in Studio. "
+            "These content types are not recommended for use in courses due to non-compliance with one or "
+            "more of the base requirements, such as testing, accessibility, internationalization, and documentation."
```

Comment:
@cahrens @lamagnifica @sstack22 I'd like to suggest a couple of small tweaks to this Help text. Second sentence replace "These" with "Unsupported'.
Unless it has already been discussed and rejected, I'd also suggest replacing "content types" with "problems and tools", in line with terms we use in the title.
"...you can add unsupported problems and tools. ... Unsupported problems and tools are not recommended..."